spec/requests/api/v2/providers/courses/publish_spec.rb

```diff
@@ -109,18 +109,24 @@
     describe 'failed validation' do
       let(:json_data) { JSON.parse(subject.body)['errors'] }
 
-      context 'no enrichments' do
+      context 'no enrichments or sites' do
```

Comment:
can we break this into two tests, please enrichment is painful enough.

Change the original course to fulfil the original tests
then add an explicit one for the no sites.

Reply:
I read it as 'no enrichments AND no sites'

Reply:
Yeah, I don't think it's necessary to break this up. The file is already quite long...

Reply:
cool